specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/examples/P2SVpnGatewayGetConnectionHealthDetailed.json

```
@@ -0,0 +1,23 @@
+{
+    "parameters": {
+        "subscriptionId": "subid",
+        "resourceGroupName": "p2s-vpn-gateway-test",
+        "virtualNetworkGatewayName": "p2svpngateway",
```

Think this is your model validation error
```suggestion
        "gatewayName": "p2svpngateway",
```